src/BlockTypes/ProductRatingCounter.php

```diff
@@ -0,0 +1,212 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+/**
+ * ProductRatingCounter class.
+ */
+class ProductRatingCounter extends AbstractBlock {
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'product-rating-counter';
+
+	/**
+	 * API version name.
+	 *
+	 * @var string
+	 */
+	protected $api_version = '2';
+
+	/**
+	 * Get block supports. Shared with the frontend.
+	 * IMPORTANT: If you change anything here, make sure to update the JS file too.
+	 *
+	 * @return array
+	 */
+	protected function get_block_type_supports() {
+		return array(
+			'color'                  =>
+				array(
+					'text'                            => true,
+					'background'                      => false,
+					'link'                            => false,
+					'__experimentalSkipSerialization' => true,
+				),
+			'typography'             =>
+				array(
+					'fontSize'                        => true,
+					'__experimentalSkipSerialization' => true,
+				),
+			'spacing'                =>
+				array(
+					'margin'                          => true,
+					'padding'                         => true,
+					'__experimentalSkipSerialization' => true,
+				),
+			'__experimentalSelector' => '.wc-block-components-product-rating',
```

Comment:
In case one of the previous comment about more unique `className` is valid, it would've to be updated as well here. Ignore this comment if `className` from Product Rating is used on purpose 🙌